x-pack/plugins/security_solution/public/detections/components/alerts_table/timeline_actions/alert_context_menu.tsx

```diff
@@ -112,10 +114,13 @@ const AlertContextMenuComponent: React.FC<AlertContextMenuProps & PropsFromRedux
   const refetchAll = useCallback(() => {
     if (timelineId === TimelineId.active) {
       refetchQuery([timelineQuery]);
+      if (routeProps.pageName === 'alerts') {
+        refetchQuery(globalQuery);
+      }
```

Comment:
Nice! I had explicitly tried closing an alert from within timeline elsewhere in the app to see if we'd still refetch, haha! 😅 

🚀 🥮